addons/mail/static/src/js/client_action.js

```diff
@@ -302,7 +344,21 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
         this.selected_message = null;
     },
 
+    initMobileView: function() {
```

Comment:
missing docstring. also, this should be in the private section. please rename this function _initMobileView